app/code/Magento/OfflinePayments/Model/Purchaseorder.php

```diff
@@ -46,11 +48,29 @@ class Purchaseorder extends \Magento\Payment\Model\Method\AbstractMethod
      *
      * @param \Magento\Framework\DataObject|mixed $data
      * @return $this
-     * @throws \Magento\Framework\Exception\LocalizedException
+     * @throws LocalizedException
      */
     public function assignData(\Magento\Framework\DataObject $data)
     {
         $this->getInfoInstance()->setPoNumber($data->getPoNumber());
         return $this;
     }
+
+    /**
+     * Validate payment method information object
+     *
+     * @return $this
+     * @throws LocalizedException
+     * @api
+     */
+    public function validate()
+    {
+        parent::validate();
+
+        if (empty($this->getInfoInstance()->getPoNumber())) {
+            throw new LocalizedException(__('Purchase order number is a required field.'));
```

Comment:
Can you add this string to the .csv so it can be translated?

Reply:
Done.

Reply:
Thanks @miguelbalparda ! I totally forgot about this 🤦‍♂️

Reply:
All good here.